app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -134,10 +138,27 @@ class BrowserFragment : IWebViewLifecycleFragment(),
                 isReloadingForYoutubeDrawerClosed = true
                 reload()
             }
+            NavigationEvent.PIN_ACTION -> {
+                this@BrowserFragment.url?.let { url ->
+                    if (VAL_CHECKED == value) {
+                        CustomTilesAccessor.getInstance(context).pinSite(context, url)
+                        showCenteredToast(R.string.notification_pinned_site)
+                    } else {
```

Comment:
nit: I'd explicitly check `VAL_UNCHECKED == ...` and throw in the else case because we are comparing against a String